docs/test-cloud/preparing-for-upload/xcuitest.md

```diff
@@ -17,7 +17,7 @@ The steps necessary to prepare an app and its corresponding test suite for uploa
 
 ## Requirements
 
-* XCode >= 9.0
+* Xcode >= 9.0
 * Sierra or High Sierra
```

Comment:
include Mojave or just: Sierra or newer

Reply:
@john7doe this PR was moved to the private repo - this one's closed.